src/ocean_component.cpp

```diff
@@ -611,6 +630,7 @@ void OceanComponent::record_state(double time)
     annualflux_sumHL_ts.set(time, annualflux_sumHL);
     annualflux_sumLL_ts.set(time, annualflux_sumLL);
     lastflux_annualized_ts.set(time, lastflux_annualized);
+    C_IO_ts.set(time, inter.get_carbon());
 
```

Comment:
@bpbond 

Here I turned one of the tvectors into a series so that I could return them as ouptut using fectchvars, is this how we want to do it? Or is there some what to return the series of values from the vector?

Reply:
This is unfortunate but seems like the right way to go.

Reply:
yea okay messy but good to know